tests/test-collection/antsibull-nox.toml

```
@@ -6,3 +6,20 @@
 
 [sessions.lint]
 run_mypy = false
+
+[[sessions.ee_check.execution_environments]]
+name = "minimal"
+test_playbooks = ["tests/test-collection/ee.yml"]
+
+[[sessions.ee_check.execution_environments]]
+name = "fedora-toolbox"
+description = "Testing EE builds with the fedora toolbox"
+test_playbooks = ["tests/test-collection/ee.yml"]
```

This won't work, since `ee.yml` is in the collection root and not in a subdirectory `tests/test-collection` of the collection root (which would be `tests/test-collection/tests/test-collection/` in the repository):
```suggestion
test_playbooks = ["ee.yml"]
```